src/EditorFeatures/CSharpTest/UseNullPropagation/UseNullPropagationTests.cs

```diff
@@ -478,5 +478,125 @@ void Main(S? s)
 }
 ");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseNullPropagation)]
```

Comment:
:bulb: Also add a `WorkItem` attribute to each of the new tests:

```csharp
[WorkItem("https://github.com/dotnet/roslyn/issues/23043")]
```

Reply:
I think it should be ```[WorkItem(23043, "https://github.com/dotnet/Roslyn/issues/22699")]``` (:

Reply:
Ugh yes. Except to make me feel a bit better you changed the link by accident 😆 Third try?

```
[WorkItem(23043, "https://github.com/dotnet/roslyn/issues/23043")]
```